workos/webhooks.py

```diff
@@ -34,19 +48,23 @@ def request_helper(self):
 
     DEFAULT_TOLERANCE = 180
 
-    def verify_event(self, payload, sig_header, secret, tolerance=DEFAULT_TOLERANCE):
-        if payload is None:
-            raise ValueError("Payload body is missing and is a required parameter")
-        if sig_header is None:
-            raise ValueError("Payload signature missing and is a required parameter")
-        if secret is None:
-            raise ValueError("Secret is missing and is a required parameter")
-
+    def verify_event(
+        self,
+        payload: WebhookPayload,
+        sig_header: str,
+        secret: str,
+        tolerance: Optional[int] = DEFAULT_TOLERANCE,
+    ) -> Webhook:
         Webhooks.verify_header(self, payload, sig_header, secret, tolerance)
-        event = json.loads(payload, object_pairs_hook=OrderedDict)
```

Comment:
Got rid of this and just relying on pydantic's JSON parser (looks like [jiter](https://docs.rs/jiter/latest/jiter/))